iothub/device/src/Convention/ClientPropertyCollection.cs

```diff
@@ -28,10 +28,13 @@ public class ClientPropertyCollection : IEnumerable<ClientProperty>
         /// </summary>
         /// <remarks>
         /// Use the <see cref="AddRootProperty(string, object)"/> and/or <see cref="AddComponentProperty(string, string, object)"/> methods
-        /// to add properties into the collection.
+        /// to add properties into the collection. Use the <see cref="AddWritableClientPropertyAcknowledgement(WritableClientPropertyAcknowledgement)"/>
```

Comment:
Suggest putting the info about responding to a writable property in a different paragraph to break it up.

Reply:
sure.